stacks_queues/stack.py

```diff
@@ -21,13 +22,21 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+
+        try:
+            first = self.store.get_first()
+            self.store.delete(first)
+            return first
```

Comment:
Nice approach, but I don't think anything in here actually raises. My guess is you're looking for an `EmptyListError`, in which case, we can try
```suggestion
            return self.store.remove_first()
```
and we should list `EmptyListError` explicitly in the `except` clause.